### PR TITLE
1113 detailed logs user event

### LINKED ISSUE #1113: [TASK] Add log to printout more details when failed to publish a user event.
### Description

Need to print more information about the submission failure.

in publish_user_event function, log the status of error code and print out error content.
https://github.com/rokwire/events-manager/blob/develop/utilities/user_utilities.py#L335

### Acceptance Criteria
_Please describe what it means for this issue to be considered complete._

### PR DESCRIPTION
**Pull request by minump**
## Description
Added status code to all error logs in user utilities

**Fixes #1113**

[comment]: # (This project only accepts pull requests related to open issues.)
[comment]: # (If suggesting a new feature or change, please discuss it in an issue first.)
[comment]: # (If fixing a bug, there should be an issue describing it with steps to reproduce.)

## Review Time Estimate
_Please give your idea of how soon this pull request needs to be reviewed by selecting one of the options below. This can be based on the criticality of the issue at hand and/or other relevant factors._

[comment]: # (To select an option, please put an 'x' in the applicable box.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] Immediately
- [x] Within a week
- [ ] When possible

## Type of changes
_Please select a relevant option:_

[comment]: # (To select an option, please put an 'x' in the applicable box.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] Bug fix (non-breaking change which fixes an issue).
- [x] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [ ] Other (any another change that does not fall in one of the above categories.)

## Checklist:
_Please select all applicable options:_

[comment]: # (To select your options, please put an 'x' in the all boxes that apply.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] I have signed the Rokwire Contributor License Agreement ([CLA](https://rokwire.org/rokwire_cla)). (_Any contributor who is not an employee of the University of Illinois whose official duties include contributing to the Rokwire software, or who is not paid by the Rokwire project, needs to sign the CLA before their contribution can be accepted._)
- [x] I have updated the [CHANGELOG](../CHANGELOG.md).
- [ ] I have read the [Contributor Guidelines](../CONTRIBUTING.md).
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] My change requires updating the documentation.
- [ ] I have made necessary changes to the documentation.
- [ ] I have added tests related to my changes.
- [ ] My changes generate no new warnings.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] Any dependent changes have been merged and published in downstream modules.

---

[comment]: # (Template credit: This pull request template is based on Embedded Artistry {https://github.com/embeddedartistry/templates/blob/master/.github/PULL_REQUEST_TEMPLATE.md}, Clowder {https://github.com/clowder-framework/clowder/blob/develop/.github/PULL_REQUEST_TEMPLATE.md}, and TalAter {https://github.com/TalAter/open-source-templates} templates.)

### CODE REVIEW

**Comment by minump** [comment]
Base changed ot master for easier reivewing..
base branch will be changed to develop once the release branch in merged.

File: utilities/user_utilities.py [hunk 1]
```diff
@@ -329,7 +330,7 @@ def approve_user_event(objectId):
         "$set": {"eventStatus": "approved"}
     })
     if not result:
-        __logger.error("Approve event {} fails in approve_event".format(id))
+        __logger.error("Approve event {} fails in approve_event. Status code {}.".format(id, result.status_code))
```
Comment (bingzhang) [inline utilities/user_utilities.py:333]: can you also log the failure reason from the response as well?

Comment (minump) [reply utilities/user_utilities.py]: response.reason will give the text associated with the status code. Eg: OK for 200, Not Found for 404. Maybe logging the response.text would give more info?

Comment (minump) [reply utilities/user_utilities.py]: Added response.text in logs.. 
Let me know if response.reason should be included as well.

Comment (minump) [reply utilities/user_utilities.py]: Eg: Logging a successful PUT request
`__logger.info(
                    "Event {} PUT submission successful. Response {} {}".format(eventId, result.status_code, result.text))`

Would result in the below logs
```
2023-09-19T15:22:24.783Z INFO    [Thread-118] : user_utilities.py - Event 64f8eeac23a760827149006e PUT submission successful. Response 200 {
  "id": "64f8eeca940572fe64e2acb4",
  "message": "[PUT]: event id 64f8eeca940572fe64e2acb4, nUpdate = 1 ",
  "status": 200
}
```

**Review by bingzhang — APPROVED** [review]

**Pull request merged**